build-tools/pnpm-lock.yaml

```diff
@@ -8103,10 +8099,19 @@ snapshots:
       jju: 1.4.0
       resolve: 1.22.8
 
+  '@microsoft/tsdoc-config@0.17.1':
+    dependencies:
+      '@microsoft/tsdoc': 0.15.1
+      ajv: 8.12.0
+      jju: 1.4.0
+      resolve: 1.22.8
+
   '@microsoft/tsdoc@0.14.2': {}
 
   '@microsoft/tsdoc@0.15.0': {}
 
+  '@microsoft/tsdoc@0.15.1': {}
```

Comment:
Would recommend `pnpm dedupe` too, so we don't end up with entries for the latest patch and also older versions.

Reply:
Doesn't seem to have had an impact.

Reply:
I do see the 0.15.0 version gone. Something else might be depending on 0.14.2, but the changes in commit b9abe4a9aa4c43bc2fc2f74c479d1903b2151f41 are in line with what I was expecting to see, across many deps.

Reply:
I was apparently looking at a stale diff.